azkaban-common/src/main/java/azkaban/executor/ExecutionRampDao.java

```diff
@@ -70,23 +71,19 @@ public ExecutableRampMap handle(final ResultSet resultSet) throws SQLException {
             ExecutableRamp.createInstance(
                 resultSet.getString(1),
                 resultSet.getString(2),
-                ExecutableRamp.Metadata.createInstance(
-                    resultSet.getInt(3),
-                    resultSet.getInt(4),
-                    resultSet.getBoolean(5)
-                ),
-                ExecutableRamp.State.createInstance(
-                    resultSet.getLong(6),
-                    resultSet.getLong(7),
-                    resultSet.getLong(8),
-                    resultSet.getInt(9),
-                    resultSet.getInt(10),
-                    resultSet.getInt(11),
-                    resultSet.getInt(12),
-                    resultSet.getBoolean(13),
-                    resultSet.getInt(14),
-                    resultSet.getBoolean(15)
-                )
+                resultSet.getInt(3),
+                resultSet.getInt(4),
+                resultSet.getBoolean(5),
+                resultSet.getLong(6),
+                resultSet.getLong(7),
+                resultSet.getLong(8),
```

Comment:
Can you please change reference using index to column name? This is very risky and error prone.

Reply:
will do in a seperate PR.